diff --git a/node_modules/react-native-sqlite-2/RNSqlite2.podspec b/node_modules/react-native-sqlite-2/RNSqlite2.podspec
index fc23e5b..8bda54f 100644
--- a/node_modules/react-native-sqlite-2/RNSqlite2.podspec
+++ b/node_modules/react-native-sqlite-2/RNSqlite2.podspec
@@ -15,6 +15,6 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.library      = "sqlite3"
 
-  s.dependency "React"
+  s.dependency "React-Core"
 end
 
diff --git a/node_modules/react-native-sqlite-2/ios/RNSqlite2.m b/node_modules/react-native-sqlite-2/ios/RNSqlite2.m
index 809c110..f8a6fba 100644
--- a/node_modules/react-native-sqlite-2/ios/RNSqlite2.m
+++ b/node_modules/react-native-sqlite-2/ios/RNSqlite2.m
@@ -1,5 +1,5 @@
 #import "RNSqlite2.h"
-#import "sqlite3.h"
+#import <sqlite3.h>
 
 @implementation RNSqlite2
 
